SharedUtils/src/main/java/com/braintreepayments/api/Json.java

```diff
@@ -1,8 +1,11 @@
 package com.braintreepayments.api;
 
+import androidx.annotation.RestrictTo;
+
 import org.json.JSONObject;
 
-class Json {
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
It seems like some of these restriction are 💥 some of the Unit Tests

Reply:
resolved the broken tests 👍